pkg/services/ngalert/tests/util.go

```diff
@@ -3,11 +3,15 @@ package tests
 import (
 	"encoding/json"
 	"fmt"
-	"math/rand"
 	"testing"
 	"time"
 
 	"github.com/grafana/grafana/pkg/services/secrets/database"
+	"github.com/grafana/grafana/pkg/util"
```

Comment:
Ideally, it should be local imports first and then external imports at the bottom but not all files follow the same rule.

As long we at least group them in the two groups (local vs external) I'm happy. Do you mind organising them a bit?